pkgs/development/interpreters/racket/variant.patch

```diff
@@ -0,0 +1,54 @@
+diff --git racket/collects/setup/variant.rkt racket/collects/setup/variant.rkt
```

Comment:
does the following patch work for you?
```diff
--- racket/collects/setup/variant.rkt
+++ racket/collects/setup/variant.rkt
@@ -6,7 +6,8 @@
 
 (provide variant-suffix)
 
-(define plain-variant
+(define plain-variant 'cs)
+#;(define plain-variant
   (delay/sync
    (cond
      [(cross-installation?)

```

Reply:
I think I could, this should save some patch lines :)
Will check results later when build finish